gherkin-utils/java/gherkin-utils/src/main/java/gherkin/utils/formatter/GherkinPrettyFormatter.java

```diff
@@ -0,0 +1,101 @@
+package gherkin.utils.formatter;
+
+import gherkin.utils.formatter.model.BackgroundStepContainer;
+import gherkin.utils.formatter.model.IStepContainer;
+import gherkin.utils.formatter.model.ScenarioStepContainer;
+import io.cucumber.messages.Messages.GherkinDocument;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class GherkinPrettyFormatter {
+    public static String pretty(GherkinDocument gherkinDocument) {
```

Comment:
To allow graceful changes in the future these methods should not be static. Rather the formatter should be used as 

``
new GherkinPrettyFormatter().format(gherkinDocument)
```

This will allow us to add in the future:

``
new GherkinPrettyFormatter(formatterOptions).format(gherkinDocument)
```

Reply:
Sr, this is my mistake. I have already fixed.